docs/html/reference/pip_show.rst

```diff
@@ -25,10 +25,50 @@ Options
 .. pip-command-options:: show
 
 
+Format
+======
+
+The various fields in the output and their  explanation is as follows. Note that the keys in JSON format are lowercased.
+The fields below are present in the METADATA file, which follows the specification
+laid down in `PEP-241`_, `PEP-314`_ and `PEP-345`_, with the same name as the fields, unless otherwise noted
```

Comment:
For core metadata specification, I think we should point to https://packaging.python.org/specifications/core-metadata/ which offers a consolidated document.

Reply:
Have added the link to the document.